docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -6,7 +6,7 @@ Creating a ``bash`` Development Shell
 
 If an existing conda (:miniforge:`Miniforge<>`, :miniconda:`Miniconda<>`, :anaconda:`Anaconda<>`, etc.) installation is available and writable, step 1 may be skipped.
 
-.. include:: ../../shared/miniforge_instructions.rst
+.. include:: /shared/miniforge_instructions.rst
```

Comment:
Many of the doc changes below follow this pattern: Use the relative-to-doc-root absolute reference to shared files, which are simpler and easier to maintain.